back/app/controllers/web_api/v1/ideas_controller.rb

```diff
@@ -184,6 +193,10 @@ def set_idea
   end
 
   def idea_attributes
+    project = @idea&.project || Project.find(params.dig(:idea, :project_id))
+    custom_form = project.custom_form || CustomForm.new(project: @project)
```

Comment:
`@project` is not defined. Probably is has to be:

```suggestion
    custom_form = project.custom_form || CustomForm.new(project: project)
```